components/core/Routes/Download.py

```diff
@@ -0,0 +1,168 @@
+        if p is not None:
+            p.terminate()
+            p.join()
+            jsonreq = json.dumps({'jsonrpc':'2.0', 'id':'qwer', 'method':'aria2.pauseAll'})
+            jsonreq = jsonreq.encode('ascii')
+            c = urllib.request.urlopen('http://localhost:6800/jsonrpc', jsonreq)
+            if verbose:
+                print(c)
+        if not p.is_alive():
+            return '{"status":"success"}'
+        else:
+            return '{"error":"error"}'
+    except Exception as e:
+        return '{"error":"' + e.message + '"}',400
+
+
+@server.route('/api/download', methods=['POST'])
+def add_download_request():
+    token = token_validator(request.headers['token'])
+    if token is not None:
+        data = request.get_json(force=True)
+        try:
+            if check_if_bandwidth_exceeded(g.user.userName):
+                resp = Response(response='{"quota":"exceeded"}', status=400)
+            else:
+                newDownload = Download(data['link'], g.user.userName)
+                status = add_download(newDownload)
+                if status == "success":
+                    resp = Response(response='{"status":"'+ status + '"}', status=200)
+                else:
+                    resp = Response(response='{"error":"' + status + '"}', status=400)
+        except Exception as e:
+            resp = Response(response='{"error":"' + e.message + '"}', status=400)
+        resp.headers['token'] = token
+        resp.headers['Access-Control-Expose-Headers'] = 'token'
+        return resp
+    elif token is not None:
+        return '{"error":"not authorized"}', 403
+    else:
+        return '{"error":"token error"}', 403
+
+
+@server.route('/api/download/<int:id>', methods=['DELETE'])
+def remove_download_request(id):
+    token = token_validator(request.headers['token'])
+    if token is not None:
+        try:
+            status = remove_download(id, g.user.userName)
+            if status == "success":
+                resp = Response(response='{"status":"'+ status + '"}', status=200)
+            else:
+                resp = Response(response='{"error":"' + status + '"}', status=400)
+        except Exception as e:
+            resp = Response(response='{"error":"' + e.message + '"}', status=400)
+        resp.headers['token'] = token
+        resp.headers['Access-Control-Expose-Headers'] = 'token'
+        return resp
+    elif token is not None:
+        return '{"error":"not authorized"}', 403
+    else:
+        return '{"error":"token error"}', 403
+
+
+@server.route('/api/download/rate/<int:id>', methods=['POST'])
+def rate_download_request(id):
+    token = token_validator(request.headers['token'])
+    if token is not None:
+        data = request.get_json(force=True)
+        try:
+            status = rate_download(id, g.user.userName, data['rate'])
+            if status == "success":
+                resp = Response(response='{"status":"'+ status + '"}', status=200)
```

Comment:
` resp = Response(response='{"status":"'+ status + '"}', status= (200 if status == "success" else 400))`

possible to replace the if/else block to a single line like this? Applicable for several places.

Reply:
This change has been implemented in the latest commit @rehrumesh